include/flecs/addons/cpp/mixins/entity/builder.hpp

```diff
@@ -501,6 +530,17 @@ struct entity_builder : entity_view {
         return to_base();  
     }
 
+     /** Set component, mark component for auto-overriding.
+      * @see override(flexs::id_t id)
+      *
+      * @tparam T the component to set and for which to add the OVERRIDE flag
+      */
+     template<typename T>
+     Self& set_override_second(const T& val) {
```

Comment:
Methods with `_second` operate on pairs, which this method doesn't. You're missing a `flecs::entity_t first` argument. Additionally, to make sure `set` works in all cases (by value, by ref, move ref etc) a number of overloads have to be implemented, see `set_override` for examples.

Reply:
are all of those overloads needed?